requests-for-comments/2023-10-31-onboarding-emails.md

```diff
@@ -22,50 +22,52 @@ This campaign triggers when a user completes the `user signed up` event for the
 3. Check users' role in organization, using the `role_at_organization` property.
     1. If `founder` then wait 12 hours, and send Joe's Newsletter Invite email.
         1. Check if user clicked the subscription CTA.
-            1. If yes, add `newsletter_cta_clicked: true` to user.
-            2. If no, do nothing. 
+            1. If `Yes`, add `newsletter_cta_clicked: true` to user.
+            2. If `No`, do nothing. 
         2. Wait 12 hours
     2. If `engineering` then wait 12 hours, and send Andy's Newsletter Invite email. 
         1. Check if user clicked the subscription CTA.
-            1. If yes, add `newsletter_cta_clicked: true` to user.
-            2. If no, do nothing. 
+            1. If `Yes`, add `newsletter_cta_clicked: true` to user.
+            2. If `No`, do nothing. 
         2. Wait 12 hours
     3. If `product` then wait 12 hours. 
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Product Teams email.
+            1. If `Yes`, send NEW email promoting the user interview survey template
+            2. If `No`, send Advice for Product Teams email.
         3. Wait 12 hours.
     4. If `marketing` then wait 12 hours. 
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Marketing Teams email.
+            1. If `Yes`, send NEW email promoting the dashboard templates. 
+            2. If `No`, send Advice for Marketing Teams email.
         3. Wait 12 hours.
     5. If `sales` then wait 12 hours. 
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Sales Teams email.
+            1. If `Yes`, send NEW email promoting the tutorials section.
```

Comment:
<img width="668" alt="Screenshot 2023-11-09 at 13 38 28" src="https://github.com/PostHog/meta/assets/84011561/9154c3f0-c70b-4b78-8ba4-ff5900f983aa">

Reply:
- First sentence doesn't need to say PostHog twice

Reply:
<img width="584" alt="Screenshot 2023-11-09 at 16 31 01" src="https://github.com/PostHog/meta/assets/84011561/743777f3-b3da-43d1-8749-615f95b89551">